sdk/search/azure-search-documents/samples/sample_indexer_datasource_skillset.py

```diff
@@ -0,0 +1,141 @@
+    Further, we create an azure cognitive search datasource which we require to finally
+    create an indexer.
+
+    Additionally, we will also use skillsets to provide some AI enhancements in our indexers.
+
+    Once we create the indexer, we run the indexer and perform some basic operations like getting
+    the indexer status.
+
+    The datasource used in this sample is stored as metadata for empty blobs in "searchcontainer".
+    The json file can be found in samples/files folder named patient_data.json has the metdata of
+    each blob.
+USAGE:
+    python sample_indexer_datasource_skillset.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_SEARCH_SERVICE_ENDPOINT - the endpoint of your Azure Cognitive Search service
+    2) AZURE_SEARCH_API_KEY - your search API key
+    3) AZURE_STORAGE_CONNECTION_STRING - The connection string for the storage blob account that is
+    being used to create the datasource.
+"""
+
+import os
+import datetime
+
+service_endpoint = os.getenv("AZURE_SEARCH_SERVICE_ENDPOINT")
+key = os.getenv("AZURE_SEARCH_API_KEY")
+connection_string = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
+
+from azure.core.credentials import AzureKeyCredential
+from azure.search.documents import (
+    DataSource, DataContainer, DataSourceCredentials, Index, Indexer, SimpleField, edm,
+    EntityRecognitionSkill, InputFieldMappingEntry, OutputFieldMappingEntry, Skillset,
+    CorsOptions, IndexingSchedule
+)
+from azure.search.documents import SearchServiceClient
+
+
+service_client = SearchServiceClient(service_endpoint, AzureKeyCredential(key))
+
+def _create_index():
+    name = "patient-index"
+
+    # Here we create an index with listed fields.
+    fields = [
+        SimpleField(name="agebracket", type=edm.String, filterable=True, sortable=True),
+        SimpleField(name="backupnotes", type=edm.String),
+        SimpleField(name="contractedfromwhichpatientsuspected", type=edm.String),
+        SimpleField(name="currentstatus", type=edm.String, filterable=True),
+        SimpleField(name="dateannounced", type=edm.String),
+        SimpleField(name="detectedcity", type=edm.String),
+        SimpleField(name="detecteddistrict", type=edm.String),
+        SimpleField(name="detectedstate", type=edm.String),
+        SimpleField(name="estimatedonsetdate", type=edm.String),
+        SimpleField(name="gender", type=edm.String, filterable=True),
+        SimpleField(name="nationality", type=edm.String),
+        SimpleField(name="notes", type=edm.String),
+        SimpleField(name="patientnumber", type=edm.String, key=True, filterable=True),
+        SimpleField(name="statecode", type=edm.String),
+        SimpleField(name="typeoftransmission", type=edm.String),
+    ]
+    cors_options = CorsOptions(allowed_origins=["*"], max_age_in_seconds=60)
+
+    # pass in the name, fields and cors options and create the index
+    index = Index(
```

Comment:
`SearchIndex`. See #10860.

Reply:
Yes I think it is best to merge this PR, then do the updates in the other PR?

Reply:
10860 should be done in a different PR